nova_vm/src/ecmascript/builtins/control_abstraction_objects/promise_objects/promise_constructor.rs

```diff
@@ -221,12 +230,118 @@ impl PromiseConstructor {
         Ok(Promise::resolve(agent, arguments.get(0)).into_value())
     }
 
+    fn r#try(agent: &mut Agent, this_value: Value, arguments: ArgumentsList) -> JsResult<Value> {
```

Comment:
**nitpick:** Especially now that this function's documentation is not available in the ECMAScript spec directly, it'd be nice to have the header / doc comment for the function link to the existing spec.